packages/stratimikos/test/CMakeLists.txt

```diff
@@ -169,33 +169,72 @@ IF (${PROJECT_NAME}_ENABLE_Tpetra)
   ASSERT_DEFINED(Tpetra_INST_INT_INT)
 ENDIF()
 
-IF (${PACKAGE_NAME}_ENABLE_Ifpack2 AND ${PACKAGE_NAME}_ENABLE_ThyraTpetraAdapters AND Tpetra_INST_DOUBLE)
+IF (${PACKAGE_NAME}_ENABLE_Ifpack2 AND ${PACKAGE_NAME}_ENABLE_ThyraTpetraAdapters)
 
   TRIBITS_COPY_FILES_TO_BINARY_DIR(Stratimikos_cp
     SOURCE_FILES stratimikos_jacobi.xml stratimikos_jacobi_half.xml
     )
 
   TRIBITS_ADD_EXECUTABLE(
-   GaleriDriver
-   SOURCES galeri_driver.cpp
-   COMM serial mpi
-   )
+    GaleriDriver
+    SOURCES
+    galeri_driver.cpp
+    COMM serial mpi
+    )
 
- TRIBITS_ADD_TEST(
-   GaleriDriver
-   NAME "Galeri_Jacobi"
-   ARGS "--xml=stratimikos_jacobi.xml"
-   NUM_MPI_PROCS 4
-   )
+  IF (Tpetra_INST_DOUBLE)
 
- IF (Tpetra_INST_FLOAT)
-   TRIBITS_ADD_TEST(
-     GaleriDriver
-     NAME "Galeri_Jacobi_HalfPrecision"
-     ARGS "--xml=stratimikos_jacobi_half.xml"
-     NUM_MPI_PROCS 4
-     )
- ENDIF()
+    TRIBITS_ADD_TEST(
+      GaleriDriver
+      NAME "Galeri_double_Jacobi"
+      ARGS "--scalarType=double --xml=stratimikos_jacobi.xml"
+      NUM_MPI_PROCS 4
+      )
+
+    IF (Tpetra_INST_FLOAT)
+      TRIBITS_ADD_TEST(
+        GaleriDriver
+        NAME "Galeri_double_Jacobi_HalfPrecision"
+        ARGS "--scalarType=double --xml=stratimikos_jacobi_half.xml"
+        NUM_MPI_PROCS 4
+        )
+    ENDIF()
+
+  ENDIF()
+
+  IF (Tpetra_INST_FLOAT)
+
+    TRIBITS_ADD_TEST(
+      GaleriDriver
+      NAME "Galeri_float_Jacobi"
+      ARGS "--scalarType=float --xml=stratimikos_jacobi.xml"
+      NUM_MPI_PROCS 4
+      )
+
+  ENDIF()
+
+  ASSERT_DEFINED(Teuchos_ENABLE_COMPLEX)
+  IF (Tpetra_INST_COMPLEX_DOUBLE AND Teuchos_ENABLE_COMPLEX)
+
+    TRIBITS_ADD_TEST(
+      GaleriDriver
+      NAME "Galeri_complex_double_Jacobi"
+      ARGS "--scalarType=\"complex<double>\" --xml=stratimikos_jacobi.xml"
+      NUM_MPI_PROCS 4
+      )
```

Comment:
We do see this `std::complex<double>` version of the test running in [this query](https://testing.sandia.gov/cdash/queryTests.php?project=Trilinos&filtercount=3&showfilters=1&filtercombine=and&field1=testname&compare1=65&value1=Stratimikos_&field2=testname&compare2=63&value2=Galeri_complex&field3=buildstarttime&compare3=84&value3=now) showing:

<html>
<body>
<!--StartFragment-->

Site | Build Name | Test Name | Status | Time | Proc Time | Details | Build Time | Processors
-- | -- | -- | -- | -- | -- | -- | -- | --
ascic158 | PR-10507-test-Trilinos_pullrequest_gcc_7.2.0_debug-5607 | Stratimikos_Galeri_complex_double_Jacobi_MPI_4 | Passed | 1s 190ms | 4s 760ms | Completed | 2022-05-10T18:21:36 MDT | 4
ascic166 | PR-10507-test-Trilinos_pullrequest_gcc_8.3.0-7593 | Stratimikos_Galeri_complex_double_Jacobi_MPI_4 | Passed | 300ms | 1s 200ms | Completed | 2022-05-10T18:21:45 MDT | 4

<!--EndFragment-->
</body>
</html>